SECURITY-INSIGHTS.yml

```
@@ -0,0 +1,66 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2025-02-13T10:10:09.000Z'
+  last-updated: '2024-02-13'
+  last-reviewed: '2024-02-13'
+  commit-hash: 4dbf78ebc006ee5f668c0a74876ef8d6db9485be
+  project-url: https://github.com/ossf/S2C2F-attestation-schema-and-tool
+  project-release: '0.0.0'
+  changelog: https://github.com/ossf/S2C2F-attestation-schema-and-tool/changelog.md
+  license: https://github.com/ossf/S2C2F-attestation-schema-and-tool/blob/main/LICENSE
+project-lifecycle:
+  status: active
+  roadmap: https://github.com/ossf/S2C2F-attestation-schema-and-tool/roadmap.md
```

You main want to correctly point the branch in the URL or you will get a URL not found. So `main` or perhaps another branch?

```suggestion
  roadmap: https://github.com/ossf/S2C2F-attestation-schema-and-tool/blob/main/ROADMAP.md
```